src/collection-proxy.js

```diff
@@ -361,6 +361,17 @@ class CollectionProxy {
     )
   }
 
+  /**
+   * Returns the max value in the collection.
+   *
+   * @returns {Integer}
+   */
+  max () {
+    return this.all(
+      this._enqueue('size')

```

Comment:
Please enqueue the `max` method name.

```suggestion
      this._enqueue('max')
```

**Some background on the internals:**

The collection proxy allows chainable calls to run multiple functions in series. Collecting all methods in a call chain is required to process async callbacks. At some point, the chain needs a `.all()` call to retrieve the result.

The collection proxy will then call the related methods in the collection class. The collection class has the actual implementation, the proxy just ensures the order and any steps necessary to keep the collections immutable.

Reply:
Done! I am sorry for my mistake.